doc/OVA/1.0.4/CHANGELOG-1.0.4.md

```diff
@@ -0,0 +1,156 @@
+- Fix the S3 sanity cleanup issues. 
+- Fixed the issue where the NextMarker is not getting properly set in list-objects using boto3 when the bucket contains more than 1000 objects. 
+- Fixed the issue where object listing with prefix specified continues key enumeration even after prefix match stops. 
+- Fixed the bucket policy allows access when both deny and allow permission is present. 
+- The s3_bundle_generate.sh script displays the error "Repository 'csm_uploads': Error parsing config: Error parsing "baseurl = '/3rd_party'": URL must be http, ftp, file or https not ""
+- Fixed the issue of handle probable delete index entry delete while s3server failure during object writes.
+- Added the fsync call to the put_keyval operation.
+- CSM CLI/GUI list 500 users.
+- [Splunk] Fixed the Ceph S3 regression in list object API. 
+- [Splunk] Fixed the issue of failure for s3tests.functional.test_s3.test_bucket_acls_changes_persistent. 
+- [Splunk] Fixed the issue of s3tests.functional.test_s3.test_bucket_notexist. 
+- [Splunk] Fixed the issue of bad_amz_date_epoch test cases on the bucket and object creation. 
+- [Splunk] Fixed the issue of s3tests.functional.test_s3:test_bucket_create_naming_bad_punctuation fails.
+- [Splunk] Fixed the issue of s3server crash in s3tests.functional.test_s3.test_multipart_upload_empty. 
+- [Splunk] Fixed the s3tests.functional.test_s3:test_bucket_create_naming_bad_punctuation issue in S3 Auth Server. 
+- [Splunk] Fixed the test_object_requestid_matchs_header_on_error issue. 
+- Fixed the issue of calling the UploadPart operation the Service is unavailable. 
+- [Splunk] Fixed the s3tests.functional.test_s3.test_multipart_upload fails issue when it ran against main branch. 
+- [Splunk] Fixed the error code mismatch for aws4 bad_ua and bad_auth date cases issue. 
+- Updated the HA Proxy.
+-	Fixed the issue where the Motr HA interface using the wrong tag.
+-	Fixed the issue of consul state is not updating for process failure.
+-	Added the license header for Motr data recovery (motr_data_recovery.sh).
+-	Update the BE error message.
+-	Fixed the issue where the Motr Kernel service is not starting on the remote node.
+-	Updated the message description for data inconsistent.
+-	Updated the IEM alert message.
+-	Optimized the object listing performance.
+-	Updated the SSPL support bundle temporary directory path.
+-	Fixed the issue of URL decode 'continuation-token' during List Object V2 request validation.
+-	Fixed the OVA for the host file and set the salt-master default configuration in case of a single node.
+-	Fixed the issue where the unboxing script is failing when the static IP network configuration option is selected.
+-	Fixed the issue where the S3 CLI requests fail with "No route to host" error intermittently.
+-	Updated the SSL certificates.
+-	Fixed the issue where SFTP session returns with non-zero exit code.
+-	Updated the glusterfs service relation with the salt-master.
```

Comment:
Updated the glusterfs service relation with the salt-master.

Such things can be filtered out as users wont be able to understand or make sense out of this.